commands/template_store_pull.go

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) OpenFaaS Author(s) 2018. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+package commands
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	templateStorePullCmd.PersistentFlags().StringVarP(&templateStoreURL, "url", "u", DefaultTemplatesStore, "Use as alternative store for templates")
+	templatePull, _, _ := faasCmd.Find([]string{"template", "pull"})
+	templateStoreCmd.PersistentFlags().AddFlagSet(templatePull.Flags())
+
+	templateStoreCmd.AddCommand(templateStorePullCmd)
+}
+
+// templateStorePullCmd pulls templates from default store or custom store if set
+var templateStorePullCmd = &cobra.Command{
```

Comment:
For future reference - this is generally only needed on public / exported types.